src/WinRT.Runtime/Projections.CustomTypeMappings.g.cs

```diff
@@ -0,0 +1,1246 @@
+        public static void RegisterIEnumerableMapping()
+        {
+            if (FeatureSwitches.EnableDefaultCustomTypeMappings)
+            {
+                return;
+            }
+
+            if (Interlocked.CompareExchange(ref _IEnumerable, 1, 0) == 1)
+            {
+                return;
+            }
+
+            RegisterCustomAbiTypeMapping(
+                typeof(IEnumerable),
+                typeof(ABI.System.Collections.IEnumerable),
+                "Microsoft.UI.Xaml.Interop.IBindableIterable",
+                isRuntimeClass: false);
+        }
+
+        /// <summary>Registers the custom ABI type mapping for the <c>"Microsoft.UI.Xaml.Interop.IBindableVector"</c> WinRT type.</summary>
+        public static void RegisterIListMapping()
+        {
+            if (FeatureSwitches.EnableDefaultCustomTypeMappings)
+            {
+                return;
+            }
+
+            if (Interlocked.CompareExchange(ref _IList, 1, 0) == 1)
+            {
+                return;
+            }
+
+            RegisterCustomAbiTypeMapping(
+                typeof(IList),
+                typeof(ABI.System.Collections.IList),
+                "Microsoft.UI.Xaml.Interop.IBindableVector",
+                isRuntimeClass: false);
+        }
+
+        /// <summary>Registers the custom ABI type mapping for the <c>"Microsoft.UI.Xaml.Interop.INotifyCollectionChanged"</c> WinRT type.</summary>
+        public static void RegisterINotifyCollectionChangedMapping()
+        {
+            if (FeatureSwitches.EnableDefaultCustomTypeMappings)
+            {
+                return;
+            }
+
+            if (Interlocked.CompareExchange(ref _INotifyCollectionChanged, 1, 0) == 1)
+            {
+                return;
+            }
+
+            RegisterCustomAbiTypeMapping(
+                typeof(INotifyCollectionChanged),
+                typeof(ABI.System.Collections.Specialized.INotifyCollectionChanged),
+                "Microsoft.UI.Xaml.Interop.INotifyCollectionChanged",
+                isRuntimeClass: false);
+        }
+
+        /// <summary>Registers the custom ABI type mapping for the <c>"Microsoft.UI.Xaml.Interop.NotifyCollectionChangedEventArgs"</c> WinRT type.</summary>
+        public static void RegisterNotifyCollectionChangedActionMapping()
+        {
+            if (FeatureSwitches.EnableDefaultCustomTypeMappings)
+            {
+                return;
+            }
+
+            if (Interlocked.CompareExchange(ref _NotifyCollectionChangedAction, 1, 0) == 1)
+            {
+                return;
+            }
+
+            RegisterCustomAbiTypeMapping(
+                typeof(NotifyCollectionChangedAction),
+                typeof(ABI.System.Collections.Specialized.NotifyCollectionChangedAction),
+                "Microsoft.UI.Xaml.Interop.NotifyCollectionChangedEventArgs",
```

Comment:
NotifyCollectionChangedAction

Reply:
Fixed in https://github.com/microsoft/CsWinRT/pull/1483/commits/2413d8ed1fd6034238be93057240d79caee4e6b6!